SPDX-License-Identifier: GPL-3.0-or-later
Include limits.h to use ULONG_MAX.
--- make-3.82.orig/remake.c
+++ make-3.82/remake.c
@@ -25,6 +25,7 @@ this program.  If not, see <http://www.g
 #include "debug.h"
 
 #include <assert.h>
+#include <limits.h>
 
 #ifdef HAVE_FCNTL_H
 #include <fcntl.h>
